src/themes/greyVest/index.js

```diff
@@ -923,19 +923,17 @@ export let Pager = props => (
   />
 )
 
-export let PagedResultTable = InjectTreeNode(
-  observer(({ tree, node, ...props }) => (
-    <>
-      <ExampleTypes.ResultTable tree={tree} node={node} {...props} />
-      <Flex style={{ justifyContent: 'space-around', padding: '10px' }}>
-        <Pager tree={tree} node={node} />
-      </Flex>
-    </>
-  ))
-)
+export let PagedResultTable = contexturify(({ tree, node, ...props }) => (
```

Comment:
I don't think you need to wrap this one.

Reply:
I think you're right - Result Table and Pager are already wrapped